### PR TITLE
static keyboard SVG + angle-mod support

### PR DESCRIPTION
**Pull request by fabi1cazenave**

### CODE REVIEW

**Comment by Nuclear-Squid** [comment]
It looks really cool ! Love it ! I have a few suggestions though

**Comment by fabi1cazenave** [comment]
Looks like you forgot to mention your suggestions in your review. :-)

**Review by Nuclear-Squid — COMMENTED** [review]
Finally found this stupid button, sorry for the wait

File: js/demo.js [hunk 1]
```diff
@@ -2,27 +2,50 @@ window.addEventListener('DOMContentLoaded', () => {
...
+    keyboard.clearStyle()
+    document.body.classList.remove('demo');
+    demo.hidden = true;
+  }
+  button.onclick = open;
+  demo.onclick = (event) => {
+    if (event.target.id === 'demo') {
+      close();
+    }
   };
```
[the remark this answers is not in the document]
Comment (fabi1cazenave) [reply js/demo.js:49]: Oh, you’re totally right. I took that part from the Lafayette website which is **very** conservative, but in 2024 `<dialog>` is safe to use.

File: css/demo.css [hunk 2]
```diff
@@ -1,13 +1,60 @@
...
+}
+
+#demo {
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  background-color: #8888;
```
[the remark this answers is not in the document]
Comment (fabi1cazenave) [reply css/demo.css:38]: Same reason : copied and pasted. Thanks for the suggestion.

**Comment by Nuclear-Squid** [comment]
Nice !

**Pull request merged**